### ISSUE TITLE
Enforcer cannon module name changed, breaking imported loadouts from EDMC and similar tools

### ISSUE THREAD

**Issue by ArghArgh200** [EDCD/coriolis#62]
Removed Enforcers from the ship and retried, and it worked.

`Hpt_MultiCannon_Fixed_Small_Strong` seems to be the new module's name.

Error:
```Browser: Mozilla/5.0 (Windows NT 6.3; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/55.0.2883.87 Safari/537.36
Path:
Error: Unknown
Details:
scriptUrl: /import?data=H4sIAPQwjVgC_-Vc227bSBL9Fz97gu7qe94m8eSym9kdxMEMFoOBQEttmxuJ1FJUJt4g_76nSckWb-LFSibBJkBgKmKxuvtU1TndRX86m0fZTXr29BN-WEfzOL87e8o-n59dZ96fPb2OlhuPi61fhq-sojipflUQvpv5jc8--Or_8CdcfMZ_xouzp47Oz1bpYrv0m_ClH7NVus0Kg8WH4afK88I9nCyTTpjy1vg69llxs09u4sT77PUFns4Ycbb7PjO88tXfP50l0QpGw4ezVbTZnJ2f5XdrfIIvfoiWW_zEnggpFUnSUgilxefzyl0Lf-2Tud9bnd36aJnfzlbbZd5qjIwiYYWTBo7LHmM3y_QqWs5ut8tll8Uf2BMmrRHKkpCWYL7Hpv-4Xqab-IOvWqSKRQYXgy0mjXW2b8z5rc9W8fyoQcLcwaaV2jlHn__YL_e76CasBNEfASXzeO3DsmFptRFayvOzzTLNXycL__HsqYQjez9e-IXPojxOk9nzFNDKcz8rUTN7mUULL-BICijm2RZwWWdxmpWg2z_mjf8QIKv21_8o7e5svPLRh7uLLe44P9smyzRKPBDE7sfEpTBOa8ECgJ-lV1dLz3g_Xo0gxel-EOWNs8v8fiBxuow3s9_izHf73-bPoR-E-fzj_kpUrmTlSlWudOXKVK5s5codXnFWXl34ebQcOAek7ueguG32yzK689nLLN2uZz8ukAXCYkaLaMIsFAbpG_FD_PV-_FSkw-fpssioYaVeZHjU5W18nV9kcZmVe9KslpzM8TTL2S7NSi3stDSrhGXGCmY5q2exMq225z9uFKKROTK40dRuXONxIdmt0z991vpcbjQzhmljFTGq3X69WczSdR6v4o0_6js5YtY54Tivu1C5q5oWMWmGG-e4cpUh5yGFFQ8PhXW2RpL993a1Lj7BVLRl2VAHdJFiudaM2UaK5botxbpqilUHKfZ1ks9e4SnZIqBkdhn_1-vZ8yVGozpgyHqS64vLi9mbNLl5GyU3viuzClRIjYWAIy8w-HdR8n4QQoViFdf3NxeOq9JxMS5-gEXHgyOvtjf-VZQt1mmc5HwQL5HK9vAS_cBL7LGI-dNHa1SHRbSKillrgZ-1JFGoGUMM1SG8uz2a58fjQDhllXHWwRum243Ml_F6tsF8tkciEWgIajxKvHQdJmrDqABYIJCddpprQiS133-LZUj8ZjNbY67mcXLTboppIThHgClElzj0ZbMGJJFio22ezrDmi4P5EA8wbMbPuW0JIFCkSgBx_RBAr9b57Gdwovh5lCRw_UX8EUkkYKkDhtQTP7-VU_DPDxg55uEGaO0IIsaY06wBXTo9dI15FHSlJKUBG6yU1XIidCUx7ZRRHFlUymnQxbi1kS5wXjkeucj_CB4tJBK6E41MXsfu_rorlXOtjJWEoEZZov9f8L4J3xubeDUfiF6ltH4keEEamIM25Fy7qeBVhqPocREcU5PAy5H6lUUt1wgj3UBfICB1BlXNuwVjgAkEkFCPRi9zmFhhHYNqVsa0wHeeZlkpQje3frk8IYTJdkP4ZbwKcroA1ZcGsXRcm4I7vImv_eV2vU6zfBjT1lThMQf3H1IZNY7KcJJcgJrCoZ-jOClFwWaQQyxwxIHUn8sjAVXeVsJxvs12WxDxehl3UXOssCFMpLak66Gxs1ZBYTWuLFg1l84aBMZoWcCclUo7EYaP7D5Uj4RocuWfwD1Eu9MPU9CxpQNNggeTUE7JAOmxES2xLEJqhcRAoLBDnaipEwEDhjuBIq90MyBNm6BQpiooqCooSuAVQDYlkGmgmBC1UNxZuoiz7l0aKDKMoAC9X8Tb1ehSwgaXEvPIUgIKowWXUG_SqKmlhAXt57jDwOW0UkISVdEwggqD5n90KQCtCuYkiJUQZjyOAScuoOwRxsaIr0aEQrz3VJESUF-8jCAJsRb80snxa_UjJSgomwWTgQYlPVWCGhsIuBaQBF36r5fHExGHDEYI2J5dpArjhupUiBwL8UsT-D_0qhMIXehX9fXUpmDfHE7jZZxH2d2w3XBDxlXKw8_FDW99nFyn2dyvfFIhPTSO9MC-E7biFn1Tbv0CKpb_Lb0atEssDnaJ9zfO2s5Bfo2vsgge_gvUOv1zwpbxL0t8HKbrx_U6S6P57eU2zgfsE2tD4JiVmStNVe2McggSonApJI2LeJNn8RVCKRvEXIn3bVrbPXMVSh3Jf0XKWjw8fX8yh5n3nWeGmsBbkRR4g3x2WCtDrEPioQYiSZOSThnqM7i52-R-VTdYyVgGKo1AWxgZIRuF-dDYji6WxnpNF8TRIB8SZwAtyGMzt7mW1IbMWT3sc1XiWAdAEX-2VwsdTW8Nm6_im9sXmf_P1ifzQCrLb_-6m7R2gALymodD0j1KA-SHST2mhx-qSNOHz3V47rGDDigZkHnGuSISYw5YWPkHJIzxVuyVj-6UYgBCONZxihN4hu62UHW-ygS1EAQd6CCrbYsYaSuXRquqGOEtmCqfPAFNvA1NxeLPnqUpoqRzg0tYAEZxGSDzNlpEA7OZkhXvL32ySbPNoesjCxHkBgld6KNLzBLjpS4b5o3r6bvgtOeWzLEj2N3cxn65GLErQFYJZ5hCepW2zhJ31nYZtbtpAhTRIkkbZaB3OoxcZel7n8wyf-OTDrZpoJcQm46Yc7bdSpJmq1Aquq0ADwaBIQ1j-Lfdik98dnNXnAR2WQqDgtACAXVKWDLtho7PcyXkuJMyFFHJw8bOiNFVAxdIME4jQ3KilmNJ1RK4tlYM6rsIl8XDX4ZJifa1wByP3j6qWzd5T_Y6KW_YnDFS30cPjYge5LKWAT0HYXu2P7LsGY_oOLIEJLl4CGnxLToly8PkYU6R6XNKT3JKCGuBy3un1FCngjoQvHrWDN6Q--xZdHfoEB_nEFfWhmPPvUO6lBgDaLe2SELDu-DUpPYMDFqjiitiQ5vgosWi1VQ4dyEdmh5Eo1liWgscEpTEulhmQpveeO8qCUuhwITQJuek4U1qXJo7MHP_UVvVqeVCA81hBFeoPc1cSKyNxPBaLjS1Fg1MzRRlejQbNo32dcSh4ijG7sFrvhR4xRTwyh2DVU6zxlnAWPQilQEXTCkGTnsa-MIgSa5Ru53SI9qHrAvtQ0IoyFd7ii5NxDiTFpWAW6kn4lN_D_i0hRdyID7lYHy6Y_isL8l7GMnrS1JBhlAa7FlaHdoT1JQVrvBDIg1NFIJASTetUbhqD5zOMWPCRoI4ReMxLDKuLXRm4ICPTOTg04Ij3kuUDkQzVXdYNRuCZnlqNP-9REaXgNSQLQ9Ydl8Ky5P6OCHSOeQZC6cL5nGpVpIWnCMpQqadKNUya0A7CMQ9cFI9oZu_2mYmwtFoaODVlrg6KVlAiEJkKxCaIC8nJmP3l8C3t32-AmDOCi_EEABrQzR4z6z21d_rrb1xkod903nQe9dRaCacLeNV3J7sUGJJOCYEyjSTE_oJBFmgxYXjWWmafcaHzmS7Btl2ACO4CNxDc97w46idKnKVIwvCjBgjKZvokm2H-6a6n6Zr3cIvLi9eHzy9WNTj2kz27KnB4uF4fvq4jvDoLkmupTGSFScrl6touRxz3K8NWJga2vcoHnfYL0J2FBoyQDMx9bDfMGnAG8PGLNHEw36wTgPZaaF1jvevdZzw_1Cc0avQ_B66d3Wj16SbvfLQsEZWhlMNd6Ij03M56HSf97U5FuiZXeZZWvQXf9l2R245b0KWTgxZ4o-DLAv95eGQGwp2aqvjxJYUjscqwYwO7811taSUvh91pZL_kPSQKyDkSAo7oWdSMXgD6gu2I5j4qvClbxK-7-LkblTCBRkVPcKf7Wu54EaOEFb7ve7OtzDBqwSIlVGcN9q9O5gtPQGrJdwnoNSNGt4QyCUUOiYJY-YNTdNPHAh6nMKOmUKqVoNur-ZnBRWkFVNIFpz1Md79ccMgsSbDaxaGQ6ICp00KIdoIqq2901lrOS-3d8vzs5JBsFPs6-8N9hFTEtaWxLQCZzoxnMkebRfsWJQefV8UdMHDWzzWGsiwRy11hbsqjjptjcRqi95tiJ3dns2NYFU4EyLQhq4RM8zqMbmEWAYvduH1ATehZZZpFDiBwoClHIpm5zrfUP4KaH5XAmIclsXJUzOxE6Zmses14EH0iqGp2TqkZM0h6Zvte0dTsw7raJVmpMa_Ogo9iHgA9eXaNcKtg_mGdgpiReGxeLoYtqtxHP3F3i3T4VzKQlm7QqBOS8fie0jH8uQQDtsmp2MX1mqCIgrn3g2i0Alh44gM6oLWgvgICIvw8k_o4wKhncAu8Eztwn6_0GYwgjkoUHGsHzY0ToZgrBmFMwhEU5iCaQCm7wHA6vQ5-NhWVz8MUNctiENgw6ah4UesaK1nkZTlMhyY8UY7ytGGMwseA06Jv8LSY4kBygi0I8nwKlFjbEMINHhzaEXQXBHI_1BmUMOl_Kq4fOs38SZI4HG41KfHpXwcLoUtxAsLu0V0KlwyE96s4sqFra8xuBRKSJR7Z0N6fwQNrpduhIclYySqt6bmazKVeZpd7db4-AFCofyKzk0Fq8yZibhV3wNuzaDXYRR3lbEgeeeXcfL-TbRN5reBT2-zzOezYHpcIyWFOl93yn4jTpVbOftfADPgRQNuH1za3bzd5Okq3pQvGjwL5se-VXD0Vzed3X_z01k4wAvNUApcwDB78HvBJFQBQpds5SGfP_8PtsKjd6NMAAA%3D
line: 0
col: 0
error: {}
```

**Comment by cmdrmcdonald** [comment]
Fixed with 9e330ea

**Comment by cmdrmcdonald** [comment]
Fixed in 2.2.12.
